.github/workflows/windows.yml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches:
+      - clean-meson-windows
+  pull_request:
+    branches:
+      - clean-meson-windows
+
+jobs:
+  test:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Choco python
+        uses: crazy-max/ghaction-chocolatey@v1
+        with:
+          args: install python --version=3.9.9 --no-progress
+      - name: set-python-path
+        run: |
+          # Install path, culled from output of Choco install above.
+          $py_dir = 'C:\hostedtoolcache\windows\Python\3.9.9\x64'
+          echo "$py_dir;$py_dir\Scripts;" >> $env:GITHUB_PATH
+      - name: show-python-version
+        run: |
+          python --version
+      - name: install-rtools
+        run: |
+          choco install rtools --no-progress
+          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
+      - name: show-gfortran
+        run: |
+          gcc --version
+          gfortran --version
+      - name: pip-packages
+        run: |
+          pip install numpy cython pybind11 pythran meson ninja
+      - name: openblas-libs
+        run: |
+          choco install unzip
+          pip install gdown
+          gdown --id 1G3jwxMY4ZVRqvz6owl0kmw_Innmr4ZnV
+          unzip -d c:\ openblas_32_if.zip
+          echo "PKG_CONFIG_PATH=c:\opt\openblas\if_32\64\lib\pkgconfig;" >> $env:GITHUB_ENV
+      - name: meson-configure
+        run: |
+          git submodule update --init --recursive
+          meson build --prefix=$PWD\build
+      - name: meson-build
+        run: |
+          ninja -j 128 -C build
```

Comment:
What's the reason for `-j 128`? Isn't that going to be quite slow compared to `-j 2`? Maybe a left-over test setting?

Reply:
I tried -j 64 but still got the Cython out-of-order build errors...

Reply:
Ah makes sense, you do all jobs in parallel and then the fastest one gets triggered in time. Decent workaround.